test/e2e/util.go

```diff
@@ -501,7 +501,7 @@ func awNamespacePhase(ctx *context, aw *arbv1.AppWrapper, phase []v1.NamespacePh
 }
 
 func waitAWPodsReady(ctx *context, aw *arbv1.AppWrapper) error {
-	return waitAWPodsReadyEx(ctx, aw, threeHundredSeconds, int(aw.Spec.SchedSpec.MinAvailable), false)
+	return waitAWPodsReadyEx(ctx, aw, ninetySeconds, int(aw.Spec.SchedSpec.MinAvailable), false)
```

Comment:
Cool beans, you've reverted that.

Reply:
While I think this is good, isn't this going to break the multiple-requeuing test? The idea was to requeue something 3 times before not requeuing anymore and setting `status: Failed`. I guess it didn't fail, but I'm just wondering what is going to happen to the test I'm referring to.